components/base/kinematicbase/ptgKinematics_test.go

```diff
@@ -42,6 +47,31 @@ func TestPTGKinematicsNoGeom(t *testing.T) {
 
 	fs := referenceframe.NewEmptyFrameSystem("test")
 	f := kb.Kinematics()
+
+	defaultBaseGeom, err := spatialmath.NewSphere(spatialmath.NewZeroPose(), 150., b.Name().Name)
+	test.That(t, err, test.ShouldBeNil)
+	t.Run("Kinematics", func(t *testing.T) {
+		frame, err := tpspace.NewPTGFrameFromKinematicOptions(
+			b.Name().ShortName(), logger, 0.3, 0, nil, NewKinematicBaseOptions().NoSkidSteer, b.TurningRadius == 0,
+		)
+		test.That(t, frame, test.ShouldNotBeNil)
+		test.That(t, err, test.ShouldBeNil)
+
+		test.That(t, f.Name(), test.ShouldEqual, "")
```

Comment:
Shouldnt this be equal to the shortname of the base?

Reply:
This is the shortname of the base, I can switch this if you prefer.

Reply:
Yes please that would be clearer